.gitpod.yml

```diff
@@ -0,0 +1,4 @@
+vscode:
+  extensions:
+    - mtxr.sqltools
+    - mtxr.sqltools-driver-pg
```

Comment:
Added some VSCode extensions that seemed to work well for me both locally and using GitPod.